pkgs/development/libraries/mumlib/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib, fetchFromGitHub, cmake, pkgconfig
+, boost, openssl, log4cpp, libopus, protobuf }:
+with lib; stdenv.mkDerivation rec {
+  name = "mumlib-${version}";
+  version = "git-2019-02-10";
```

Comment:
`"unstable-2019-02-10";`

Reply:
@dasJ Oops! I missed this first time around. As you likely know now the date you should supply is the date of the revision, not the date you packaged the software.

Reply:
Rebased again and fixed